File: fib40.scala
object Fibonacci {
  def fib(n: Int): Int = n match {
    case 0 | 1 => n
    case _ => fib(n -1) + fib(n-2)
  }

  def main(args: Array[String]) {
    val start = System.nanoTime();
    val fib40 = fib(40);
    val diff = System.nanoTime() - start;
    println(fib40);
    println(diff / 1000000);
  }
}

File: fib40.jl
function fib(n)
  if n < 3
    return 1
  else
    return fib(n - 1) + fib(n - 2)
  end
end

start = time()
fib40 = fib(40)
diff = time() - start
println(fib40)
println(diff * 1000) # time in ms
